build.sbt

```diff
@@ -0,0 +1,41 @@
+lazy val supportedScalaVersions = List("2.13.4", "2.12.12")
+
+ThisBuild / organization := "org.tree_annotation"
+ThisBuild / scalaVersion := "2.12.12"
+ThisBuild / name := "tao-parser"
+ThisBuild / description := "first truly universal syntax for structural communication at any scale, born of a vision of total intercommunication of all software systems"
+ThisBuild / licenses := List("MIT" -> url("https://choosealicense.com/licenses/mit/"))
+ThisBuild / homepage := Some(url("https://www.tree-annotation.org/"))
+ThisBuild / scmInfo := Some(ScmInfo(url("https://github.com/tree-annotation/tao"), "https://github.com/tree-annotation/tao.git"))

```

Comment:
`/tao` is the JS parser, so better make this `/tao-parser-scala`, `/tao-parser-scala.git`.

Reply:
Right, this is wrong link. I'll replace it.